src/drivers/mct/cime_config/config_component_e3sm.xml

```diff
@@ -795,6 +795,15 @@
     <desc>2**n relative cost of machine (DO NOT EDIT)</desc>
   </entry>
 
+  <entry id="CASE_GROUP">
+    <type>char</type>
+    <group>run_desc</group>
+    <file>env_run.xml</file>
```

Comment:
I think it should be in env_case.xml.   You can't change it without re-running create_newcase.